notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>In the Google Cloud Console, you can monitor your training job at <strong>Vertex AI &gt; Training &gt; Custom Jobs</strong>. You can open the TensorBoard for each custom training job by clicking <code><strong>OPEN TENSORBOARD</strong></code>. The TensorBoards are updated in near real time.</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>